manifests/params.pp

```diff
@@ -11,7 +11,10 @@
   $repo_proxy_password         = 'absent'
   $repo_proxy_username         = 'absent'
   $repo_release                = undef
-  $repo_url_suffix             = '12.x'
+  $repo_url_suffix             = ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '7') ? {
```

Comment:
does that render properly in puppet-strings or do you need to write it on a single line?

Reply:
neither.

"Default value: `$nodejs::params::repo_url_suffix`" is what it documents today.

Reply:
ah lol that's params.pp